bazel/execution/service.go

```diff
@@ -144,3 +144,53 @@ func execReqToScoot(req *remoteexecution.ExecuteRequest) (result sched.JobDefini
 
 	return result, nil
 }
+
+// Stub of GetOperation API - most fields omitted but returns a valid hardcoded response.
+func (s *executionServer) GetOperation(
+	ctx context.Context,
+	req *longrunning.GetOperationRequest) (*longrunning.Operation, error) {
+	log.Infof("Received GetOperation request: %s", req)
+
+	if !s.IsInitialized() {
+		return nil, status.Error(codes.Internal, "Server not initialized")
+	}
+
+	// TODO get job identified by req.Name's status
+
+	op := longrunning.Operation{}
+	op.Name = req.Name
+	op.Done = true
+
+	eom := remoteexecution.ExecuteOperationMetadata{}
+	eom.Stage = remoteexecution.ExecuteOperationMetadata_COMPLETED
+
+	// Marshal ExecuteActionMetadata to protobuf.Any format
+	eomAsPBAny, err := ptypes.MarshalAny(&eom)
+	if err != nil {
+		log.Errorf("Failed to marshal ExecuteOperationMetadata: %s", err)
+		return nil, fmt.Errorf("Failed to marshal ExecuteOperationMetadata as ptypes/any.Any: %s", err)
+	}
+	op.Metadata = eomAsPBAny
+
+	res := getOperationResponse{}
```

Comment:
Would add one more TODO about moving some of this boilerplate response struct generation into a function later